### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by thekp**
Resolves n/a

**Overall change:** _bumping dependencies in psammead-figure

**Code changes:**

- _update changelog_
- _bump version_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by jamesbhobbs — APPROVED** [review]

**Review by ryanmccombe — APPROVED** [review]

File: packages/components/psammead-figure/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.0.8   | [PR#1204](https://github.com/bbc/psammead/pull/1204) Bump dependencies |
```
Comment (thekp) [inline packages/components/psammead-figure/CHANGELOG.md]: this is wrong

Comment (ryanmccombe) [inline packages/components/psammead-figure/CHANGELOG.md]:
```suggestion
| 1.0.8   | [PR#1205](https://github.com/bbc/psammead/pull/1205) Bump dependencies |
```

**Comment by thekp** [comment]
pls no merge yet

File: packages/components/psammead-figure/CHANGELOG.md (hunk 1 above)
Comment (thekp) [reply packages/components/psammead-figure/CHANGELOG.md]: pushing changes now 🏃

**Comment by thekp** [comment]
devDependencies don't need testing.
this is the dependency bump:
# Gel Foundations Changelog


| Version | Description |
|---------|-------------|
| 3.0.3 | [PR#1155](https://github.com/bbc/psammead/pull/1155) use `psammead-test-helpers@1.0.2` |
| 3.0.2 | [PR#783](https://github.com/bbc/psammead/pull/783) Update to latest psammead-test-helpers. Update snapshots. |


# Psammead Test Helpers Changelog


| Version | Description |
|---------|-------------|
| 1.0.2   | [PR#1083](https://github.com/bbc/psammead/pull/1083) Use `jest-styled-components@6.3.3` |

**Review by dr3 — APPROVED** [review]

**Pull request merged**